plugins/processors/unpivot/sample.conf

```diff
@@ -1,6 +1,14 @@
 # Rotate multi field metric into several single field metrics
 [[processors.unpivot]]
+  ## Metric mode to pivot to
+  ## Set to "original", metrics are pivoted to the original measurement name
+  ## Set to "field" creates a new metric called the field name. With this
+  ## option the tag_key is ignored. Be aware that this could lead to metric
+  ## name conflicts!
+  # metric_mode = "original"
```

Comment:
Hmmm I'm not sure if this is intuitive. How about something like
`use_fieldname_as = "metric"` and `use_fieldname_as = "tag"?`

Reply:
thank you! that is much better, I'll make that change

Reply:
Change pushed